app/src/main/java/com/twilio/notify/quickstart/notifyapi/TwilioSDKStarterAPI.java

```diff
@@ -0,0 +1,35 @@
+package com.twilio.notify.quickstart.notifyapi;
+
+import com.twilio.notify.quickstart.notifyapi.model.Binding;
+
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.jackson.JacksonConverterFactory;
+import retrofit2.http.Body;
+import retrofit2.http.POST;
+
+public class TwilioSDKStarterAPI {
+    private static String BASE_SERVER_URL = "https://f5ab0b54.ngrok.io";
```

Comment:
Can you replace this ngrok URL with YOUR_SDK_STARTER_SERVER_URL to match the README? Thanks